nomad.yaml

```
@@ -1,18 +1,20 @@
-plugins:
-  include:
-    - "nomad_dtu_nanolab_plugin.schema_packages:sputtering"
-    - "nomad_dtu_nanolab_plugin.schema_packages:rtp"
-    - "nomad_dtu_nanolab_plugin.schema_packages:thermal"
-    - "nomad_dtu_nanolab_plugin.schema_packages:gas"
-    - "nomad_dtu_nanolab_plugin.schema_packages:instrument"
-    - "nomad_dtu_nanolab_plugin.schema_packages:target"
-    - "nomad_dtu_nanolab_plugin.schema_packages:substrate"
-    - "nomad_dtu_nanolab_plugin.schema_packages:edx"
-    - "nomad_dtu_nanolab_plugin.schema_packages:xrd"
-    - "nomad_dtu_nanolab_plugin.schema_packages:ellipsometry"
-    - "nomad_dtu_nanolab_plugin.schema_packages:xps"
-    - "nomad_dtu_nanolab_plugin.schema_packages:PL"
-    - "nomad_dtu_nanolab_plugin.schema_packages:substrate"
-    - "nomad_dtu_nanolab_plugin.schema_packages:sample"
-    - "nomad_dtu_nanolab_plugin.schema_packages:analysis"
-    - "nomad_dtu_nanolab_plugin.schema_packages:basesections"
\ No newline at end of file
+plugins:
+  include:
+    - "nomad_dtu_nanolab_plugin.schema_packages:sputtering"
+    - "nomad_dtu_nanolab_plugin.schema_packages:rtp"
+    - "nomad_dtu_nanolab_plugin.schema_packages:thermal"
+    - "nomad_dtu_nanolab_plugin.schema_packages:gas"
+    - "nomad_dtu_nanolab_plugin.schema_packages:instrument"
+    - "nomad_dtu_nanolab_plugin.schema_packages:target"
+    - "nomad_dtu_nanolab_plugin.schema_packages:substrate"
+    - "nomad_dtu_nanolab_plugin.schema_packages:edx"
+    - "nomad_dtu_nanolab_plugin.schema_packages:xrd"
+    - "nomad_dtu_nanolab_plugin.schema_packages:ellipsometry"
+    - "nomad_dtu_nanolab_plugin.schema_packages:xps"
+    - "nomad_dtu_nanolab_plugin.schema_packages:PL"

```

```suggestion
    - "nomad_dtu_nanolab_plugin.schema_packages:pl"
```

The name after `:` should point to the variable in the `__init__.py` file in `schema_packages`, which is `pl` in your case.